--- origsrc/qtcanvas3d-opensource-src-5.5.1/src/imports/qtcanvas3d/context3d_p.h	2015-10-12 23:37:01.000000000 -0500
+++ src/qtcanvas3d-opensource-src-5.5.1/src/imports/qtcanvas3d/context3d_p.h	2015-10-22 22:04:31.644834100 -0500
@@ -67,6 +67,9 @@
 #ifdef NO_ERROR // may de defined in winerror.h
 #undef NO_ERROR
 #endif
+#ifdef BOOL
+#undef BOOL
+#endif
 
 QT_BEGIN_NAMESPACE
 QT_CANVAS3D_BEGIN_NAMESPACE
